e2e/testers/graph.ts

```diff
@@ -0,0 +1,43 @@
+import { Page, expect } from '@playwright/test';
+import { Tester } from './tester';
+
+export class GraphTester extends Tester {
+
+  constructor(page: Page) {
+    super(page);
+  }
+
+  async testCreateGraph() {
+    await this.openGraphForm();
+    await this.pressNextButton();
+    await this.selectFile('182848/BATT VOLT');
```

Comment:
Should be a variable

Reply:
Also for this test how is that bin file being fetched? Where is it being saved and loaded from?

Reply:
This function is for creating the graph exclusively, with the precondition that the file is uploaded. All of the functions in here are merely pieces to be put together to write full tests in the .test.ts files